suspendapp-ktor/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+  id(libs.plugins.kotlin.jvm.get().pluginId)
+
+  alias(libs.plugins.arrowGradleConfig.formatter)
+  alias(libs.plugins.arrowGradleConfig.publish)
+  alias(libs.plugins.arrowGradleConfig.versioning)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies{
+  implementation(libs.coroutines)
+  implementation(libs.arrow.fx)
+  implementation(libs.ktor.core)
+  implementation(libs.ktor.netty)
+}
```

Comment:
```suggestion
kotlin {
  // We set up custom targets rather than use Arrow Gradle Config for Kotlin,
  // Since we don't support all targets but only subset where having this behavior makes sense.
  jvm {
    compilations.all {
      kotlinOptions.jvmTarget = "1.8"
    }
  }
  js(IR) {
    nodejs()
  }
  
  linuxX64()
  mingwX64()
  macosArm64()
  macosX64()
  
  sourceSets {
    val commonMain by getting {
      dependencies {
        implementation(libs.coroutines)
        implementation(libs.arrow.fx)
        implementation(libs.ktor.core)
      }
    }
    
    val jvmMain by getting
    val jsMain by getting
    val mingwX64Main by getting
    val linuxX64Main by getting
    val macosArm64Main by getting
    val macosX64Main by getting
    
    create("nativeMain") {
      dependsOn(commonMain)
      mingwX64Main.dependsOn(this)
      linuxX64Main.dependsOn(this)
      macosArm64Main.dependsOn(this)
      macosX64Main.dependsOn(this)
    }
  }
}
```